ddtrace/contrib/internal/celery/signals.py

```diff
@@ -181,10 +183,15 @@ def trace_after_publish(*args, **kwargs):
     if span is None:
         return
     else:
-        nodename = span.get_tag("celery.hostname")
-        if nodename is not None:
-            _, host = nodenames.nodesplit(nodename)
-            span.set_tag_str(net.TARGET_HOST, host)
+        broker_url = current_app.conf.broker_url
+        parsed_url = urlparse(broker_url)
+
+        host = ""
+        if parsed_url.scheme:
+            host += parsed_url.scheme + "://"
```

Comment:
`out.host` should contain only the host ([server.address](https://opentelemetry.io/docs/specs/semconv/attributes-registry/server/) in otel terms), so we should omit the "scheme" in this case.

Reply:
Since once of the assertions is `assert async_span.get_tag("out.host") == "memory://"`, I think we should only omit it if it doesn't stay `memory`. 

Otherwise, that span tag will be completely blank in the "memory scenario".

Reply:
> Since one of the assertions is assert async_span.get_tag("out.host") == "memory://", I think we should only omit it if it doesn't say memory.

OK, that works.

Reply:
Also, do we want to set the tag if its an empty value?